src/app/services/properties-panel.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+import { SetPropertiesModel } from '../model/canvas-model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PropertiesPanelService {
+  constructor() {}
+
+  invokeSetObjectPanelProperty: Subject<SetPropertiesModel> = new Subject();
+  invokeSetObjectPanelProperty$ =
+    this.invokeSetObjectPanelProperty.asObservable();
```

Comment:
Try sending object data from properties panel and changing the properties panel view according to the data. Avoid controlling the properties panel view from app-canvas, This creates unwanted dependencies.